chatops_deployment/ansible/roles/systemd_exporter/files/systemd-exporter.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Systemd Exporter Service
+Documentation=https://github.com/prometheus-community/systemd_exporter
+After=network-online.target
+
+[Service]
+User=systemd-exporter
+Group=systemd-exporter
+Restart=on-failure
+ExecStart=/opt/systemd-exporter/systemd_exporter \
+--web.telemetry-path="/"
+StandardOutput=append:/opt/systemd-exporter/systemd-exporter.log
+StandardError=append:/opt/systemd-exporter/systemd-exporter.log
```

Comment:
Any reason we're not using normal logging locations e.g. `/var/log/systemd-exporter` as this would be way down on the list of places to look at?